### PR TITLE
Add transpiling to ES5 with (webpack + babel)

### LINKED ISSUE #3: IE11 should be supported
By making the docs work in IE11 we will prevent users' confusions about supporting IE11 ( https://github.com/sweetalert2/sweetalert2/issues/536)

Also, it's handy to quickly test docs page with IE11 and ensure everything works in it like in other browsers.

### PR DESCRIPTION
**Pull request by limonte**
Closes #3 

The diff is big, but there are just a few actual changes:

1. move inline JS from `index.html` to `main.js`
2. add webpack + babel + necessary presets 
3. in `index.html` load dynamically `main.js` for new browsers, or `bundle.js` for other browsers

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1346,497 +1346,17 @@ <h3>Contribute</h3>
...
-    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga')
-    ga('create', 'UA-83618163-1', 'auto')
-    ga('send', 'pageview')
-  }
-</script>
-<script src="./js/main.js"></script>
-<div id="rtl-container" dir="rtl"></div>
+    $.getScript(isAsyncSupported ? './js/main.js' : './js/bundle.js')
```
[the remark this answers is not in the document]
Comment (limonte) [reply index.html]: Let's just serve `budnle.js`. If some dev will be interested in the source code they will easily find the repo. Done.

**Comment by limonte** [comment]
> Instead of using multiple presets/plugins, we should just use babel-preset-env

Done.

**Comment by zenflow** [comment]
It still needs `babel-plugin-transform-async-to-generator`?

**Comment by limonte** [comment]
> It still needs babel-plugin-transform-async-to-generator?

Actually, no need for it, removed. Thanks for noticing!

**Comment by zenflow** [comment]
@limonte I don't like that there is a built file being committed but the same thing is happening already in the `./styles` dir. If PR https://github.com/sweetalert2/sweetalert2/pull/857 works out well we can use the same approach with a `gh-pages` branch. https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/

**Pull request merged**